frontend/src/components/editor/navigation/navigation.ts

```diff
@@ -606,9 +606,9 @@ export function useCellEditorNavigationProps(cellId: CellId) {
 
   const { keyboardProps } = useKeyboard({
     onKeyDown: (evt) => {
-      // For vim mode, require Shift+Escape to exit to command mode
+      // For vim mode, require Ctrl+Escape to exit to command mode
       if (keymapPreset === "vim") {
-        if (evt.key === "Escape" && evt.shiftKey) {
+        if (evt.key === "Escape" && evt.ctrlKey) {
```

Comment:
Should this be `evt.ctrlKey || evt.metaKey`? For consistency with our other keymaps, so that on macOS it will be `cmd+Esc`. Just a thought, up to you

Reply:
I like that